tests/__tests__/mixins/labels.js

```diff
@@ -1,25 +1,44 @@
 import { ATOM_GROUP_NAME } from "../../../src/enums";
 import { getWaveInstance } from "../../enums";
+import * as THREE from "three";
 
 describe("Atom labels", () => {
     let wave, atoms, labels;
 
     beforeEach(() => {
         wave = getWaveInstance();
         const atomGroup = wave.scene.getObjectByName(ATOM_GROUP_NAME);
+        labels = wave.structureGroup.children.filter((child) => child instanceof THREE.Points);
         atoms = atomGroup.children.filter((object) => object.type === "Mesh");
-        labels = atomGroup.children.filter((object) => object.type === "Sprite");
     });
 
     test("Labels are created for every atom", async () => {
         const basisAtomsNumber = wave.structure.basis.elements.length;
+        const isAllAtomsHaveLabels = atoms.every((atom) => {
+            const atomName = atom.name.split("-")[0];
+            const labelName = `labels-for-${atomName}`;
+            const labelPointsByAtomName = labels.find(
+                (labelPoints) => labelPoints.name === labelName,
+            );
+            if (!labelPointsByAtomName) return false;
+            const positions = labelPointsByAtomName.geometry.getAttribute("position")?.array;
+            let isHaveSomeLabel = false;
+            const atomPosition = new THREE.Vector3().setFromMatrixPosition(atom.matrixWorld);
+            for (let i = 0; i < positions.length; i += 3) {
+                const x = positions[i];
+                const y = positions[i + 1];
+                const z = positions[i + 2];
+                const labelPosition = new THREE.Vector3(x, y, z);
+                const distance = labelPosition.distanceTo(atomPosition);
+                if (distance < 0.00001) {
```

Comment:
How did we decide on the cutoff?

Reply:
I found that labels are not positioned directly on the position that I am set they have a cutoff of 0.00001, for a user, it's not visible also I saw that previous tests were implemented in the same way with that cutoff